python/cuml/_version.py

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 #
 
-# Version configuration
-__version__ = "23.12.00"
+import os.path
+
+with open(os.path.join(os.path.dirname(__file__), 'VERSION')) as version_file:
+    __version__ = version_file.read().strip()
```

Comment:
I believe that using `importlib.resources` is a more widely compatible approach to accessing such resource files:

```suggestion
import importlib.resources

__version__ = importlib.resources.read_text("cuml", "VERSION").strip()
```
If we don't want to hard-code "cuml", we could also use `__name__.split(".")[0]`, but I think that might be over-engineering.

If we don't want to use `importlib`, we could also use `Pathlib` for a slightly more concise expression:
```suggestion
from pathlib import Path

__version__ = Path(__file__).with_name("VERSION").read_text().strip()
```

Reply:
Good call! Done.